service/controller/v3/resource/service/update.go

```diff
@@ -3,11 +3,29 @@ package service
 import (
 	"context"
 
+	"fmt"
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/operatorkit/controller"
 )
 
 func (r *Resource) ApplyUpdateChange(ctx context.Context, obj, updateChange interface{}) error {
+	serviceToUpdate, err := toService(updateChange)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	if serviceToUpdate != nil {
+		r.logger.LogCtx(ctx, "level", "debug", "message", "updating services")
+
+		_, err := r.k8sClient.CoreV1().Services(serviceToUpdate.Namespace).Update(serviceToUpdate)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		r.logger.LogCtx(ctx, "level", "debug", "message", "updated services")
+	} else {
+		r.logger.LogCtx(ctx, "level", "debug", "message", "no need to update services")
+	}
```

Comment:
Well, why wasn't this already implemented? Anything else we do not properly reconcile? @kopiczko @TheoBrigitte

Reply:
It was some copy paste by mr @JosephSalisbury

Reply:
yeah its same in all operators, i already have prepared PRs there a well